### ISSUE TITLE
[Screen Reader-Command Palette]:-Screen reader is not announcing the suggestions when user searches for the commands.

### ISSUE THREAD

**Issue by DasaradhaKamjula** [microsoft/terminal#7907]
**User Experience:**
Users who rely on screen reader will not be able to get the suggestions info displayed on searching for commands..

**Test Environment:**
App: Windows Terminal Preview
Feature: Command Palette
Screen Reader: Narrator
OS: Win 10 2004 Build (19041.508)

**Repro Steps:**
1.Open Windows terminal Preview app
2.Open Command Palette using shortcut (Ctrl+shift+p)
3.Search for any in command in the search field
4.Observe whether screen reader is providing any announcements or not after searching for commands.

**Actual:**
Screen reader is not announcing the suggestions info when user searches for the commands. Screen reader remains silent on searching.

**Expected:**
Screen reader should provide the results info on searching commands like 10 results found or suggestions available when there are any search results and if there are no search results then it should announce as No commands found . 

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by carlos-zamora** [comment]
@DasaradhaKamjula  This is now fixed in v1.12.10334.0. Can you please confirm?

**Comment by v-rpundir** [comment]
Hi @carlos-zamora @zadjii-msft 

Screen Reader is only announcing 'Suggestion Available' but UX of Screen Reader users enhance if it also announces suggestion count like '2 Suggestion available' 

Thanks :):

**Comment by zadjii-msft** [comment]
@v-rpundir we were concerned that localizing messages for '2 Suggestion available' might be more complicated than just formatting a message with the number of results. I can whip something up for 'one suggestion available' vs '{numResults} suggestions available', but I don't know if that'll localize super well.

In any case, do you think we could close out this A11ySev2 bug and open a new A11yUsable bug tracking adding a number to the message?

**Comment by DHowett** [comment]
@v-rpundir, due to a lack of localization support for the different plural word forms in different languages (Russian has 4, Arabic has 7) across the entire Windows platform, no other applications on the platform announce the numbers of search results available. Can you re-evaluate this request?

We would like to do it, for sure, but this seems like it is a _new_ Sev3 request.

**Comment by DHowett** [comment]
@v-rpundir Sorry to be a bother: the existence of suggestions is currently announced in conformity with other Windows products. This bug is resolved as well as we can, but it is still impacting our accessibility score negatively. I'm requesting that this out-of-SLA Sev2 be closed as fixed and replaced with a new Sev3 to track announcing the _number_ of suggestions.

**Comment by v-rpundir** [comment]
Closing this bug and will log new issue after discussing with PWD Users.
